workspaces/cli-server/src/tasks/on-demand-initial-body.ts

```diff
@@ -97,6 +97,7 @@ export class OnDemandInitialBody {
       this.events.once('completed', onCompleted);
       function onCompleted(data: any) {
         cleanup();
+        child.kill(0);
```

Comment:
should it be kill()'d or exit()'d?

Reply:
i dont think there's an exit()? its this right, https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal

i think just `child.kill()` is correct, which sends a SIGTERM by default. i'm a little confused by sending it 0, because that doesn't map to a signal.

Reply:
Oh you're right, I was looking at process, not child_process

Reply:
@acunniffe if you remove the 0 it'll send the default SIGTERM